Mathlib/Data/Matrix/Rank.lean

```diff
@@ -274,4 +275,34 @@ theorem rank_eq_finrank_span_row [LinearOrderedField R] [Finite m] (A : Matrix m
   rw [← rank_transpose, rank_eq_finrank_span_cols, transpose_transpose]
 #align matrix.rank_eq_finrank_span_row Matrix.rank_eq_finrank_span_row
 
+section MulNonsingInv
+
+/-! ### Lemmas about pre or post multiplying by an invertible matrix. -/
+
+/-- Pre (Left) multiplying by an invertible matrix does not change the rank -/
```

Comment:
```suggestion
/-- Pre (left) multiplying by an invertible matrix does not change the rank -/
```
I'd just drop usage of "pre" and "post", which aren't particularly helpful compared to left and right.